packages/bitkeep/src/index.ts

```diff
@@ -1,54 +0,0 @@
-import type { WalletInit, EIP1193Provider } from '@web3-onboard/common'
-
-let bitKeepDependencies: any = null
-let bitKeepAdapter: any = null
-
-const loadBitKeepDependencies = async (getAdapter?: boolean) => {
-  if (!bitKeepDependencies) {
-    bitKeepDependencies = await import('@bitget-wallet/web3-sdk')
-  }
-  if (getAdapter && !bitKeepAdapter) {
-    bitKeepAdapter = new bitKeepDependencies.LegacyEip1193Adapter()
-    return bitKeepAdapter
-  }
-  if (getAdapter) {
-    return bitKeepAdapter
-  }
-  return bitKeepDependencies
-}
-
-/**
- * @deprecated Please use `@web3-onboard/bitget` instead.
- */
-function bitKeep(): WalletInit {
-  if (typeof window === 'undefined') return () => null
-
-  return () => {
-    return {
-      label: 'BitKeep',
```

Comment:
Rather than removing this module, updating the displayed name is more simple for dapp devs to handle maintenance and follows our process. We can add a deprecation flag but we wont want to remove this from the repo